tests/generative/test_query_model.py

```diff
@@ -48,6 +50,7 @@
         min_value=datetime.date(1900, 1, 1), max_value=datetime.date(2100, 12, 31)
     ),
     float: st.floats(min_value=0.0, max_value=11.0, width=16, allow_infinity=False),
+    str: st.text(alphabet=["a", "b", "c"], min_size=1, max_size=3),
```

Comment:
This wouldn't allow an empty string.  Do we want to?  It'd probably be a likely value in a string column.

Reply:
Probably! I'll add the empty string and force push.

Reply:
I've added the empty string by setting `min_size=0`:

https://github.com/opensafely-core/databuilder/blob/8f8e23645b4b5aac0f210cdf1ab14fdf5a7dc5fd/tests/generative/test_query_model.py#L53

This is because [`alphabet` should be a collection of length one strings][1] and the empty string is length zero. Although `min_size=0` is the default, setting it explicitly is consistent with the other strategies.

[1]: https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.text

Reply:
yes, that's what I'd expected, thanks